functions:
    - $(find rosplan_stage_exploration_demo)/scripts/exploration_sensing_functions_sampling.py
topics:
    robot_at:
        params:
            - kenny
            - '*'
        topic: /amcl_pose
        msg_type: geometry_msgs/PoseWithCovarianceStamped
    doughnut_visible_from:
        params:
            - '*'
            - '*'
        topic: /update_visibility
        msg_type: std_msgs/String
